flutter_cache_manager/lib/src/web/mime_converter.dart

```diff
@@ -54,6 +54,7 @@ const mimeTypes = {
   'audio/aac': '.aac',
   'audio/x-aac': '.aac',
   'audio/midi audio/x-midi': '.midi',
+  'audio/x-m4a': '.m4a',
```

Comment:
[Looking at this list](http://help.dottoro.com/lapuadlp.php), the main type seems to me `audio/m4a` and x-m4a is an alias for that. I think it is better to include both.
```suggestion
  'audio/x-m4a': '.m4a',
  'audio/m4a': '.m4a',
```

Reply:
OK